Teams/04 Fundamentals of Microsoft Teams/Lab.md

```diff
@@ -231,14 +233,14 @@ Before registering the bot, note the URL configured for the solution in Visual S
 
 ### Register the bot
 
-1. Go to the Microsoft Bot Framework portal at https://dev.botframework.com and sign in. (The bot registration portal accepts a Work or School Account or a Microsoft Account.)
-1. Click Register. (If the Register button is not shown, click **My bots** in the top navigation.)
+1. Go to the Microsoft Bot Framework Portal at https://dev.botframework.com/bots/new and sign in. 
```

Comment:
Other places in this PR where this same instruction occurs, it also includes this statement:  "(The bot registration portal accepts a Work or School Account or a Microsoft Account.)"   Should it be added here as well?